sauce/features/budget/display-goal-amount/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'DisplayTargetGoalAmount',
+  type: 'checkbox',
+  default: false,
+  section: 'budget',
+  title: 'Display Target Goal Amount And Overspend Warning',
+  description: 'Displays the target goal amount for every category with a goal, and warns you when you spend over your goal.'
```

Comment:
I'd point out that a new column will be added to the display titled 'Goal' in which the goal amounts will be shown.